IowaPD_people.yaml

```diff
@@ -668,16 +674,16 @@ associationDefinitions:
     fqn: 'ol.involvedin'
     entitySetName: 'JohnsonCounty_IowaCity_CFSInvolvedIn'
     src: "JohnsonCounty_IowaCity_Vehicle"
-    dst: "JohnsonCounty_IowaCity_Dispatch"
+    dst: "JohnsonCounty_IowaCity_CallsForService"
     propertyDefinitions:
       ol.id:
         type: 'ol.id'
         column: 'Dis_ID'
   OfficerInvolvedIn:
     fqn: 'ol.involvedin'
     entitySetName: 'JohnsonCounty_IowaCity_CFSInvolvedIn'
-    src: "Officers"
-    dst: "JohnsonCounty_IowaCity_Dispatch"
+    src: "OfficersPersonnel"
+    dst: "JohnsonCounty_IowaCity_CallsForService"
     propertyDefinitions:
       ol.id:
         type: 'ol.id'
```

Comment:
let's take out person age from OfficerInvolvedIn, I think they're not talking about the officer's age...